igvc_gazebo/nodes/ground_truth/main.cpp

```diff
@@ -0,0 +1,108 @@
+#include <ros/ros.h>
+#include <nav_msgs/Odometry.h>
+#include <mutex>
+#include <geometry_msgs/Vector3.h>
+
+#include <tf/transform_datatypes.h>
+#include <igvc_utils/NodeUtils.hpp>
+
+ros::Publisher ground_truth_pub, diff_pub, angle_diff_pub;
```

Comment:
Minor style note here, but comma separated declarations are generally avoided as the rules around them become tricky with more complex types.